gcp_variant_transforms/options/variant_transform_options.py

```diff
@@ -262,3 +262,39 @@ def validate(self, parsed_args):
             '--variant_merge_strategy {}|{}'.format(
                 MergeOptions.MOVE_TO_CALLS,
                 MergeOptions.MERGE_WITH_NON_VARIANTS))
+
+
+class PreprocessOptions(VariantTransformsOptions):
+  """Options for preprocess."""
+  _REPORT_NAME = 'conflicts_report.csv'
+  _RESOLVED_HEADERS_FILE_NAME = 'resolved_headers.vcf'
+
+  def add_arguments(self, parser):
+    parser.add_argument(
+        '--report_all',
+        type='bool', default=True, nargs='?', const=False,
+        help=('By default, only the incompatible VCF headers will be reported. '
+              'If true, it also reports the undefined headers and malformed '
+              'recorded.'))
+    parser.add_argument(
+        '--split_alternate_allele_info_fields',
+        type='bool', default=True, nargs='?', const=True,
+        help=('If true, all INFO fields with Number=A (i.e. one value for each '
```

Comment:
Could please make the description more relevant to preprocess tool? Also we have this parameter defined somewhere else, right? If yes, why do we need to redefine it here?